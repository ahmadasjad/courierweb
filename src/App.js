import React,{useEffect,StrictMode} from 'react';
import { ColorModeScript } from '@chakra-ui/react';
import { Provider } from 'react-redux';
import {store} from './redux/store';
import {ChakraProvider} from '@chakra-ui/react';
import {Helmet,HelmetProvider} from "react-helmet-async";
import { 
  BrowserRouter
} from 'react-router-dom';
import axios from 'axios'
import WebRoutes from './routes/index.routes'
import theme from './theme/index.theme'
function App() {
 useEffect(()=>{

    axios.get("https://mak-courier-server.herokuapp.com/products/product-details/1")
    .then((res)=>{
        console.log(res)
    })
  })
  return (<StrictMode>
    <HelmetProvider>
    <ChakraProvider theme={theme}>
      
    <ColorModeScript />
   
       <Helmet>
                <meta charSet="utf-8" />
                <title>Courier GO</title>
                <meta name="keywords" content="courier,send parsel,dtdc price"/>
      </Helmet>
      <Provider store={store}>
      <BrowserRouter>
        <WebRoutes />
      </BrowserRouter>
      </Provider>
    </ChakraProvider> 
    </HelmetProvider>
    </StrictMode>
  );
}

export default App;
