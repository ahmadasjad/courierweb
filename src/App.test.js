import React from 'react';
import App from './App';
import { create } from 'react-test-renderer';
import {shallow} from 'enzyme';
////enjyme
const components=shallow(<App/>)

/////snapshot
describe('My first snapshot test',()=>{
    test('testing childs', () => {
    let tree = create(<App />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})