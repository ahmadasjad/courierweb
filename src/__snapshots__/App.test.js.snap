// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test testing childs 1`] = `<span />`;
